ci_js/doc/index.rst

```diff
@@ -0,0 +1,60 @@
+========
+ CI JS
+========
+
+
+Configuration
+=============
+
+
+Usage
+=====
+
+Создание нового тура
+--------------------
+
+* Открыть в верхнем меню ``[[ CI JS ]]``
+* В боковом меню во вкладке ``CI JS`` выбрать кнопку ``Tours``
+* Нажать на кнопку ``Create``
+* В открывшемся окне заполнить поля:
+    * ``Tour name``
+    * ``URL`` - стартовый адрес тура
+    * ``Wait for``
+    * ``Skip enabled`` - поставить галочку, если нужно добавить кнопку Skip в подсказках
+    * ``Using assets`` - выбрать нужные ассетсы( assets_frontend, assets_backend, etc.)
+* Во вкладке ``Sets`` нажимаем на ``Add an item``
+* В появившемся окне в поле ``Template`` нажимаем на выпадающее меню и выбираем подходящий шаблон.
+* Если нужного шаблона нет, то нажимаем на ``Create and Edit``:
+    * В поле ``Template name`` вводим название шаблона
+    * Во вкладке ``Steps`` нажать ``Add an item`` и в появившемся окне заполнить следующие поля:
+        * ``content`` - будет выводится в tips
+        * ``trigger`` - селектор элемента, по которому нужно кликнуть в тесте, либо напротив чего разместить указатель(обязательное поле)
+        * ``extra_trigger`` - клик на ``trigger`` сработает, когда указанный здесь элемент будет видимым.
+        * ``timeout`` - максимальное время ожидания условия
+        * ``position`` - позиция для указателя на выбор: left, rigth, top, bottom
+        * ``width`` - ширина указателя в пикселях. По умолчанию 270px
+        * ``edition`` - указание того, в каком издании odoo можно выполнять шаг (community, enterprise or any edition). По умолчанию стоит параметр any edition
+        * ``run`` - действия которые выполняются, когда тур запущен автоматически
+        * ``auto`` - шаг будет пропущен в не автоматическом запуске
+    * Если нужно добавить ещё шаг, то нажать ``Save & New``, либо ``Save & Close``, если шаги закончились.
+    * Нажать ``Save``, чтобы сохранить получившийся шаблон
+* Когда шаблон выбран, то под полем ``Template`` появляются поля с названиями переменных, которые принимает шаблон(если он их принимает), нужно заполнить эти поля, в соответствии с контекстом
+* Когда все необходимые поля заполнены, можно нажать ``Save & New``, если нужно добавить еще один set. Если составление сетов для тура закончено можно нажать ``Save & Close``
+* Чтобы сохранить получившийся тур нужно нажать ``Save``, после этого увидим наш тур во вкладке ``Tours``
+
+Отладка и быстрый запуск тура
+-----------------------------
+
+* Открыть в верхнем меню ``[[ CI JS ]]``
+* В боковом меню во вкладке ``CI JS`` выбрать кнопку ``Tours``
+* В списке туров найти необходимый и открыть его
+* Нажать на ``Start tour``, после чего тур запустится
+
+
+Генерация файла тура
+--------------------
+
+* Открыть в верхнем меню ``[[ CI JS ]]``
+* В боковом меню во вкладке ``CI JS`` выбрать кнопку ``Tours``
+* В списке туров найти необходимый и открыть его
+* Нажать на кнопку ``Download tour``
```

Comment:
@trojikman Please add a description of two buttons:
1. In the ``[[ CI JS ]] >> CI JS >> Templates`` section:
"Make XML" button - generates xml record so that you can add it to a set of frequently used templates.
2. In a tour form in each line with set:
"Generate JS" button - generates js code so that you can paste it in browser console and test it.